hudi-flink/src/main/java/org/apache/hudi/sink/compact/FlinkCompactionConfig.java

```diff
@@ -93,7 +93,7 @@
       + "There is a risk of losing data when scheduling compaction outside the writer job.\n"
```

Comment:
There is no risk anymore. We could change the description now. Async compaction scheduling is actually recommended.